### PR TITLE
PetHRRT pipeline fixes (#2)

### PR DESCRIPTION
**Pull request by laemtl**
Additional fixes that complement #9.

### CODE REVIEW

File: uploadNeuroDB/HRRT_PET_insertion.pl [hunk 1]
```diff
@@ -380,17 +381,15 @@
...
-                $message, $NeuroDB::ExitCodes::UNKNOW_PROTOCOL, $log_file
+                $message, $NeuroDB::ExitCodes::UNKNOWN_PROTOCOL, $log_file
             );
             # insert error message into notification spool table
             $notifier->spool(
                 'HRRT_PET insertion'   , $message,   0,
                 'HRRT_PET_insertion.pl', $upload_id, 'Y',
                 'N'
             );
-            exit $NeuroDB::ExitCodes::UNKNOW_PROTOCOL;
```
Comment (laemtl) [inline uploadNeuroDB/HRRT_PET_insertion.pl:391]: If missing execution is aborted despite the fact that the protocol does not seem mandatory for the pipeline to succeed.

File: uploadNeuroDB/NeuroDB/ExitCodes.pm [hunk 2]
```diff
@@ -210,5 +210,4 @@ our $HRRT_ALREADY_INSERTED          = 285; # if HRRT archive already exists in D
 our $HEADER_INSERT_FAILURE          = 286; # if could not insert matlab info into the
                                            # MINC header
 our $HRRT_ARCHIVE_INSERTION_FAILURE = 287; # if HRRT archive insertion into the
-                                           # database has failed
-our $UNKNOW_PROTOCOL                = 288; # if HRRT protocol is not found
```
Comment (laemtl) [inline uploadNeuroDB/NeuroDB/ExitCodes.pm:214]: $UNKNOW[N]_PROTOCOL already exists.

File: uploadNeuroDB/imaging_upload_file.pl [hunk 3]
```diff
@@ -199,6 +200,7 @@ =head2 Methods
     hostName     => $Settings::db[3]
 );
 $db->connect();
+$db->{mysql_auto_reconnect} = 1;
```
Comment (laemtl) [inline uploadNeuroDB/imaging_upload_file.pl:203]: Solve issues of type `MySQL server has gone away`

File: uploadNeuroDB/NeuroDB/ExitCodes.pm (hunk 2 above)
Comment (cmadjar) [reply uploadNeuroDB/NeuroDB/ExitCodes.pm:214]: I am guessing that `UNKNOW_PROTOCOL ` is not used anywhere else. Just to be sure.

File: uploadNeuroDB/HRRT_PET_insertion.pl (hunk 1 above)
Comment (cmadjar) [reply uploadNeuroDB/HRRT_PET_insertion.pl:391]: not sure I understand here. We can talk during our call.

**Review by cmadjar — APPROVED** [review]

File: uploadNeuroDB/NeuroDB/ExitCodes.pm (hunk 2 above)
Comment (laemtl) [reply uploadNeuroDB/NeuroDB/ExitCodes.pm:214]: Yes, I checked.

**Comment by cmadjar** [comment]
@laemtl Is it ready to be merged? I see you put back the exit so I would tend to think it is but just in case, can you confirm?

**Comment by laemtl** [comment]
Ready!

**Pull request merged**